app/scripts/services/import-modal.js

```diff
@@ -55,6 +55,15 @@
 
         return importService.mergeFile($scope.rootDirectory, mode.value)
           .then(function () {
+            var promises = [];
+
+            $scope.rootDirectory.forEachChildDo(function (file) {
```

Comment:
Does this mean that it would save even non-imported unsaved files?

I think I remember that forEachChildDo() is recursive in the tree right?